src/config.py

```diff
@@ -29,3 +35,12 @@ def montagu_password(self):
     @property
     def orderlyweb_url(self):
         return self.cfg["orderlyweb_url"]
+
+    def diagnostic_reports(self, group, disease):
```

Comment:
There's a good chance I've missed something here, but what happens when we trigger an unknown disease/group? It looks like that might end up silently being a noop? How do we find out that this has not really triggered anything useful?

Reply:
Ah, it looks like there is some logging, but it's not clear to me how we'd find that out? I guess something like an out-of-the-box monitoring tool (https://docs.celeryproject.org/en/stable/userguide/monitoring.html) might help?

Reply:
Yeah, haven't thought that much about how we surface the logging. It's available in the container which isn't so friendly, but getting some monitoring set up would be good. Maybe once emailing is set up we could email science if there's a config problem or report failure, or post to Teams?

Reply:
Added a placeholder ticket to remind us to do something around monitoring/notification https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-4146